doc/snippets/Microsoft.Data.SqlClient/SqlRetryLogicBase.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<docs>
+    <members name="SqlRetryLogicBase">
+        <SqlRetryLogicBase>
+            <summary>It retrieves next time interval with respect to the number of retries if transient condition happens.</summary>
+        </SqlRetryLogicBase>
+        <NumberOfTries>
+            <summary>Maximume number of retries.</summary>
+            <value>
+            <see langword="int" /> that returns the maximum number of retry execution attemps is expected to attemp after the first failure.</value>
```

Comment:
```suggestion
            <see langword="int" /> that indicates the maximum number of retry execution attemps is expected to attemp after the first failure.</value>
```
```suggestion
            <see langword="int" /> that returns the maximum number of expected retry execution attempts after the first failure.</value>
```

Do we need to link to `int`? I would suggest something like "The maximum number of..."